### PR TITLE
Update hub-namespaces.adoc

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
# Description

Update the documentation regarding the Hub XMLRPC API namespaces, according to the latest changes on it

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Uyuni

### CODE REVIEW

File: modules/large-deployments/pages/hub-namespaces.adoc [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 
-* The ``hub`` namespace is used to target the Hub XMLRPC API Server.
-It supports all the methods that are available on a {productname} Server XMLRPC API as well as Hub-specific XMLRPC endpoints.
+* The ``hub`` namespace is used to target the Hub-specific XMLRPC endpoints.
 Hub-specific methods are primarily related to authentication.
 * The ``unicast`` namespace is used to target a single peripheral server registered in the hub.
 It redirects any call transparently to one specific server and returns any value as if the server's XMLRPC API endpoint was used directly.
 * The ``multicast`` namespace is used to target multiple peripheral servers registered in the hub.
 It redirects any call transparently to all the specified servers and returns the results in the form of a ``map``.
+* It is possible to target directly the underlying XMLRPC API of the Hub server, given that the HUB XMLRPC API exposes all the methods that are available on a {productname} Server XMLRPC API.
```
Comment (moio) [inline modules/large-deployments/pages/hub-namespaces.adoc:18]: What is the point that you are trying to make here? The list is supposed to contain the three namespaces ("The Hub XMLRPC API supports three different namespaces").

Comment (chiaradiamarcelo) [reply modules/large-deployments/pages/hub-namespaces.adoc:18]: Yeah, I can't find the right words for this.

My point is that the user can authenticate to the hub XMLRPC API, using, eg, client.hub.Login("user","pass")

and then you can:
- use the hub-specific endpoints (eg, client.hub.listServerIds)
- use the multicast
- use the unicast
- AND call the Uyuni XMLRPC of the hub server directly, with the same client, and it does not require any special namespace. For example, you can have a statement like 'client.system.listSystems(hubSessionKey)', which will execute the  'system.listSystems' method of the Uyuni XMLRPC API of the Hub server

Comment (chiaradiamarcelo) [reply modules/large-deployments/pages/hub-namespaces.adoc:18]: TBH, I'm not sure if it's worth to mention this in the doc

Comment (moio) [reply modules/large-deployments/pages/hub-namespaces.adoc:18]: Let me try to reformulate...

```suggestion
* methods called without any of the above namespaces will be forwarded to the normal XMLRPC API of the Hub (ie. the one exposed on ports 80 and 443)
```

Comment (chiaradiamarcelo) [reply modules/large-deployments/pages/hub-namespaces.adoc]: thanks!

File: modules/large-deployments/pages/hub-namespaces.adoc [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
-client.unicast.[namespace].[methodname](hubKey, serverId, param1, param2)
+client.unicast.[namespace].[methodName](hubSessionKey, serverId, param1, param2)
 ----
-* You can obtain ``hubKey`` using different authentication methods.
+* The ``hubSessionKey`` can be obtained using different authentication methods.
 For more information, see xref:large-deployments:hub-auth.adoc[].
-* The ``multicast`` namespace assumes all methods receive ``hubKey``, a list of ServerIDs, then lists of per-server parameters as specified by the regular server API.
-The return value is a map, with `Successful` and `Failed` entries.
+* The ``multicast`` namespace assumes all methods receive ``hubSessionKey``, a list of ServerIDs, then lists of per-server parameters as specified by the regular server XMLRPC API.
+The return value is a map, with `Successful` and `Failed` entries for each server involved in the call.
```
Comment (keichwa) [inline modules/large-deployments/pages/hub-namespaces.adoc:32]:
```suggestion
The return value is a ``map``, with ``Successful`` and ``Failed`` entries for each server involved in the call.
```
Earlier we'd tagged the word/term "map".

File: modules/large-deployments/pages/hub-namespaces.adoc [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
 It redirects any call transparently to all the specified servers and returns the results in the form of a ``map``.
+* methods called without specifying any of the above namespaces will be forwarded to the normal XMLRPC API of the Hub (ie. the one exposed on ports 80 and 443)
 
 
 Some important considerations for hub namespaces:
 
-* Individual server IDs can be obtained using ``client.hub.listServerIds(hubKey)``.
-* The ``unicast`` namespace assumes all methods receive ``hubKey`` and serverID as their first two parameters, then any other parameter as specified by the regular Server API.
+* Individual server IDs can be obtained using ``client.hub.listServerIds(hubSessionKey)``.
+* The ``unicast`` namespace assumes all methods receive ``hubSessionKey`` and serverID as their first two parameters, then any other parameter as specified by the regular Server API.
```
[the remark this answers is not in the document]
Comment (chiaradiamarcelo) [reply modules/large-deployments/pages/hub-namespaces.adoc:26]: Yes, it is

**Pull request merged**